data/tlg2016/tlg001/tlg2016.tlg001.1st1K-lat1.xml

```diff
@@ -80,7 +80,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"?>
 	
 	<text>
 		<body>
-			<div type="edition" n="urn:cts:greekLit:tlg2016.tlg001.1st1K-lat1" xml:lang="lat">
+			<div type="translation" n="urn:cts:greekLit:tlg2016.tlg001.1st1K-lat1" xml:lang="lat">
```

Comment:
same question as above. I think the Latin is original and the Greek is a translation.